src/Faithlife.WebRequests/WebServiceRequestBase.cs

```diff
@@ -125,7 +125,7 @@ public string Method
 		public ByteRange Range { get; set; }
 
 		/// <summary>
-		/// True if HTTP redirects should not be followed automatically.
+		/// True if HTTP redirects should not be followed automatically. If <see cref="WebServiceRequestSettings.GetHttpClient"/> is set, then this property is ignored.
```

Comment:
Consider moving new comment to `<remarks>`.

Reply:
With this one (and the one below), I prefer having them in `<summary>`. It's an important enough semantic that I want it in IntelliSense.

Reply:
Fair enough.